cypress/e2e/party.cy.js

```diff
@@ -1,9 +1,41 @@
 describe('party spec', () => {
   beforeEach(() => {
     cy.intercept('GET', 'https://obscure-caverns-08355-6f81aa04bbe3.herokuapp.com/api/v1/trainers/1/pokemons/2', {
+      statusCode: 200,
+      fixture: 'single_pokemon'
+    }).as('getSinglePokemon');
+
+    cy.intercept('GET', 'https://obscure-caverns-08355-6f81aa04bbe3.herokuapp.com/api/v1/trainers/1/pokemons', {
       statusCode: 200,
       fixture: 'PkmnData'
     }).as('getParty');
+
     cy.visit('http://localhost:3000/');
-})
+    cy.get('.start-words').click()
```

Comment:
Great testing structure, very thorough. I've found the more specific you can get with cypress tests the better!